go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -507,6 +505,19 @@ func newSharedGateway(nodeName string, subnets []*net.IPNet, gwNextHops []net.IP
 		return nil, err
 	}
 
+	gw.readyFunc = func() (bool, error) {
+		// the name of the patch port created by ovn-controller is of the form
+		// patch-<logical_port_name_of_localnet_port>-to-br-int
+		patchPort := "patch-" + bridgeName + "_" + nodeName + "-to-br-int"
+		// Get ofport of patchPort
+		ofportPatch, _, err := util.RunOVSVsctl("--if-exists", "get", "interface", patchPort, "ofport")
```

Comment:
in another patch that isnt as urgent we should consolidate these get patch port commands everywhere to some utility or store it somewhere so we dont have to get it all the time

Reply:
https://paste.centos.org/view/e13aeeab